packages/tex-to-myst/src/citations.ts

```diff
@@ -34,6 +25,16 @@ function createCitation(
   } else {
     cite.label = citations[0];
   }
+  const args = getArguments(node, 'argument');
+  if (args.length > 0) {
+    // If there is only one argument, it is the suffix
+    const [prefix, suffix] = args.length === 1 ? [undefined, args[0]] : args;
+    const prefixText = texToText(prefix).trim();
+    const suffixText = texToText(suffix).trim();
+    const citation = isGroup ? cite.children?.[0] : cite;
```

Comment:
This feels a little funny if there are multiple citations and prefix/suffix, but maybe this is just the expected behaviour and not a problem.

Reply:
Yep, you are correct. This should be on the first and last child of the group:

![image](https://user-images.githubusercontent.com/913249/228034501-afbe779a-5c48-41aa-95b4-0497d5006cd5.png)

![image](https://user-images.githubusercontent.com/913249/228034471-04a37917-ee8b-4132-85b9-139f58e52bcb.png)

Will fix that up.

Reply:
Ok, pushed. I think there are actually quite a few intricacies when you get into textual/narrative citations. Where the prefix is repeated, but not the suffix. I am going to leave as is for now, and fix when we get complaints. Lots of weird work-arounds on the internet for this type of stuff.

![image](https://user-images.githubusercontent.com/913249/228037818-7aac0edd-2f9b-4da4-a1b6-86149d06c073.png)

![image](https://user-images.githubusercontent.com/913249/228037778-618c3c7e-08a1-4484-9636-f4eae5e5c2fc.png)